### ISSUE TITLE
[Feature] Backbone pretrain config

### ISSUE THREAD

**Issue by TUDelftHao** [zcablii/Large-Selective-Kernel-Network#7]
### What's the feature?

Hi, thank you for your amazing work!

I would like to know if you can share the lsknet backbone pre-train config for the ImageNet, like the batch size, optimizer, lr scheduler, learning rate, etc. I want to finetune the backbone and modify the architecture for downstream applications. 

Thank you in advance!

**Comment by zcablii** [comment]
My lsknet backbone pre-train config is:
8 GPU, DROP_PATH=0.1  -b 128 --lr 1e-3 --drop-path $DROP_PATH --amp --epochs 300 --model-ema
Based on this code-base: https://github.com/Visual-Attention-Network/VAN-Classification

**Comment by TUDelftHao** [comment]
> 

Thank you for your kind reply! I will have a try!

**Comment by TUDelftHao** [comment]
> My lsknet backbone pre-train config is: 8 GPU, DROP_PATH=0.1 -b 128 --lr 1e-3 --drop-path $DROP_PATH --amp --epochs 300 --model-ema Based on this code-base: https://github.com/Visual-Attention-Network/VAN-Classification

Thank you for your kind reply! I will have a try!
